deploy/kubernetes/helm/che/templates/deployment.yaml

```diff
@@ -100,7 +100,7 @@ spec:
           valueFrom:
             secretKeyRef:
               key: ca.crt
-              name: {{ .Values.global.tls.secretName }}
+              name: {{ .Values.global.tls.selfSignedCertSecretName }}
```

Comment:
It is not good to change the value name.
Somebody might use it

Reply:
No, I didn't change the name. This change just says: "Read `ca.crt` from selfSignedCertSecretName secret instead of secretName secret". Names are unchanged.